src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationGeneratorConfiguration.java

```diff
@@ -3659,4 +3659,27 @@ public List<NotificationGenerator> updateTribunalDecisionRule31AipNotificationGe
                 )
         );
     }
+
+    @Bean("appellantRefundRequestedAipNotificationGenerator")
+    public List<NotificationGenerator> updateTribunalDecisionRule31AipNotificationGenerator(
+        AiPAppellantRefundRequestedNotificationEmail aipAppellantRefundRequestedNotificationEmail,
```

Comment:
Can you check with Ben and see if we need to notify admin? We are currently sending out email to Admin for this email in LR journey, maybe we need to notified Admin too for AIP journey.